osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceCalculator.cs

```diff
@@ -84,35 +86,52 @@ private double computeDifficultyValue(ScoreInfo score, TaikoDifficultyAttributes
                 difficultyValue *= 1.025;
 
             if (score.Mods.Any(m => m is ModHardRock))
-                difficultyValue *= 1.050;
+                difficultyValue *= 1.10;
```

Comment:
Since the changes here already account for hit windows properly, and does not concern SV, why does hard rock specifically need to be buffed here?

Reply:
This was done in balancing, as mid-range accuracy with HR was pretty underweighted, in saying that though @Natelytle could the model be fit for greater accuracy leniency on super high OD's?

Reply:
Most likely not without increasing complexity and decreasing estimation accuracy, I think a HR multiplier buff is a better direction if HR in particular is underweight